sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -339,6 +339,27 @@ class ResolveSessionCatalog(
         v1TableName.asTableIdentifier,
         partitionSpec)
 
+    case ShowColumnsStatement(table, namespace) =>
+      val sql = "SHOW COLUMNS"
+      val v1TableName = parseV1Table(table, sql).asTableIdentifier
+      val resolver = conf.resolver
+      val db = namespace match {
```

Comment:
Why is this done here? Wouldn't this be done inside `ShowColumnsCommand` anyway?

Reply:
I have move it because this comment. 
https://github.com/apache/spark/pull/26182#discussion_r339897559
It seems simpler to check all in one place